python/test/regression/test_cast_matmul.py

```diff
@@ -65,18 +70,28 @@ def matmul_kernel(A, B, C, M, N, K,  #
 
 @pytest.mark.parametrize("M, K, N, w_dtype, x_dtype, out_dtype",
                          [(M, K, N, w, x, o)  #
-                          for (M, K, N) in [(128, 128, 128), (1280, 768, 1024)]  #
+                          for (M, K, N) in [(128, 128, 128), (768, 768, 1024)]  #
```

Comment:
Which of these is exercising the repeated K dimension elements bug?

Reply:
It's not a bug. Just that when accumulating a lot of int8s will result in inf. What's your recommendation?

Reply:
Hmm, just noted that it's on the M dimension but not K...

Reply:
I guess I can just skip int8 for the large matrix?

Reply:
Another way is probably further limiting the value range to [0, 1]

`torch.randint(0, 2, shape, device=device, dtype=dtype)`